### PR TITLE
Add preference to toggle gyroscope control

### PR DESCRIPTION
**Pull request by johnshaughnessy**
Adds a preference to prevent the gyroscope from controlling the camera view

### CODE REVIEW

**Comment by cvanmeurs** [comment]
Thanks for adding this.  

Might also be nice if it were disabled by default in landscape mode, when the joysticks are enabled, because they fight with each other.

File: src/assets/locales/en.json [hunk 1]
```diff
@@ -311,6 +311,7 @@
   "preferences.movementSpeedModifier": "Movement speed modifier",
   "preferences.enableOnScreenJoystickLeft": "Enable left on-screen joystick for moving around",
   "preferences.enableOnScreenJoystickRight": "Enable right on-screen joystick for looking around",
+  "preferences.enableGyro": "Enable gyroscope (when supported by browser/device",
```
Comment (takahirox) [inline src/assets/locales/en.json:314]: Lack of the closing bracket.

Comment (johnshaughnessy) [reply src/assets/locales/en.json]: oops! fixed.

**Review by takahirox — APPROVED** [review]
The change looks good to me. And I confirmed on my Samsung S8+ that this change works fine.

**Pull request merged**